packages/cli/src/bin.ts

```diff
@@ -54,9 +54,11 @@ process.removeAllListeners('warning')
 const debug = Debug('prisma:cli')
 process.on('uncaughtException', (e) => {
```

Comment:
About this it means that error code will switch to 0, which is confusing

https://nodejs.org/api/process.html#event-uncaughtexception

> The 'uncaughtException' event is emitted when an uncaught JavaScript exception bubbles all the way back to the event loop.
> **By default**, Node.js handles such exceptions by printing the stack trace to stderr and exiting with code 1, overriding any previously set [process.exitCode](https://nodejs.org/api/process.html#processexitcode_1).
> **Adding a handler for the 'uncaughtException' event overrides this default behavior.**
> Alternatively, change the [process.exitCode](https://nodejs.org/api/process.html#processexitcode_1) in the 'uncaughtException' handler which will result in the process exiting with the provided exit code. Otherwise, **in the presence of such handler the process will exit with 0.**